config-templates/ibm/ibm-csi-block/1.4.0/storage-class-parameters.json

```diff
@@ -0,0 +1,60 @@
+[
+  {
+	"description": "Storage class name to be created",
+	"displayname": "Storage Class name",
+	"name": "sc-name",
+	"required": "true",
+	"category": "text"
+  },
+  {
+	"description": "Space efficiency of volume to be created",
+	"displayname": "Space efficiency type",
+	"name": "space-efficiency",
+	"required": "false",
+	"category": "text"
+  },
+  {
+	"description": "Pool ID of volume to be created in",
+	"displayname": "Pool ID",
+	"name": "pool",
+	"required": "true",
+	"category": "text"
+  },
+  {
+	"description": "Existing secret name",
+	"displayname": "Secret name",
+	"name": "secret-name",
+	"required": "true",
+	"category": "text"
```

Comment:
Replace `"category": "text"` to  `"category": "config"

We have 2 categories  `config` and `secret`

Reply:
in every `"category": "text"` in file?

Reply:
yes

Reply:
in `custom_parameters.json` too?